platform/src/main/resources/config/anchor-config-default-values.yaml

```diff
@@ -139,17 +139,33 @@ event_processor:
   context_path: /
   # The listening port of the event processing server
   port: 8084
+  # The flag to enable the status callback to the client domain
+  client_domain_status_callback_enabled: true
+  # The flag to enable the event delivery to the anchor business server
+  callback_api_event_delivery_enabled: true
   # The queue where the event processor will
   queue:
     type: kafka
     kafka:
       bootstrap_server: localhost:29092
-      event_type_to_topic:
-        all: dev_ap_event_single_queue
-        quote_created: ap_event_quote_created
-        transaction_created: ap_event_transaction_created
-        transaction_status_updated: ap_event_transaction_status_changed
-        transaction_error: ap_event_transaction_error
+
+# The list of clients for the Anchor server to safely communicate with the outside client domains.
+# Each item in the list may contain the following fields:
+#   - domain: the domain of the client
+#   - type: the type of the client. The supported values are `custodial` and `non-custodial`
+#   - public_key: the public key of the client. This must match the public key of the TOML file of the client domain
```

Comment:
I suggest naming this something more specific in case we want to add public keys for sending & receiving payments. We should also design the structure to support multiple keys just in case. For example:

```
wallets:
  - domain: whitebit.com
    auth_accounts:
      - GBI2IWJGR4UQPBIKPP6WG76X5PHSD2QTEBGIP6AZ3ZXWV46ZUSGNEGN2
    source_accounts:
      - GA22WORKYRXB6AW7XR5GIOAOQUY4KKCENEAI34FN3KJNWHKDZTZSVLTU
    destination_accounts:
      - GA22WORKYRXB6AW7XR5GIOAOQUY4KKCENEAI34FN3KJNWHKDZTZSVLTU
```

I don't think we should add source / destination accounts right now, but I think its good to prepare for them if requested by a partner.

Reply:
I avoided `wallets` because it actually represents `clients` and this section is to identify the client attribution.

Reply:
The purpose of this section is to identify the client (wallet). The detail (`source_accounts`, `destination_accounts`, etc) of the wallet should be obtained from the TOML file from `https://domain/.well-known/stellar.toml` to avoid multiple sources of truth and avoid unnecessary inconsistency.

Reply:
The challenge there is that custodial wallets may not have toml files.

Reply:
We can have accounts etc to be optional, but I agree in general toml file is preferred.

Reply:
What's the purpose of 
`source_accounts: - GA22WORKYRXB6AW7XR5GIOAOQUY4KKCENEAI34FN3KJNWHKDZTZSVLTU`
and `destination_accounts: - GA22WORKYRXB6AW7XR5GIOAOQUY4KKCENEAI34FN3KJNWHKDZTZSVLTU`

Reply:
They don't have a functional purpose today, but MoneyGram whitelists custodial wallets source & destination accounts today, so I could see them requesting this.

Reply:
Adding configuration won't break the backward compatibility. IMO, we may skip these two fields until requested.